### PR TITLE
Feat-responsive

### LINKED ISSUE #13: [Style] : Responsive issue
There is responsive issue at https://pop-shop-github.vercel.app/home/shop

Screenshot 2024-05-11 104723

### PR DESCRIPTION
**Pull request by Sid-80**
## Describe 
Every page is responsive

## Issue
resolves #13 

## Screenshots

image

image

image

image

image

image

image

image

image

image

image

image

image

### CODE REVIEW

**Comment by Sid-80** [comment]
@pooranjoyb Sorry for the inconvenience. Please review it

File: src/pages/Home/Home.tsx [hunk 1]
```diff
@@ -32,20 +32,20 @@ function Home() {
             </div>
           </div>
         </div>
-        <img className="w-1/3" src="./images/hero.png" alt="" />
+        <img className="w-1/3 py-4 sm:p-0" src="./images/hero.png" alt="" />
```
Comment (pooranjoyb) [inline src/pages/Home/Home.tsx:35]: This hero image looks very small in mobile devices :/

File: src/pages/Home/Home.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ function Home() {
   return (
     <>
       {/* Hero  */}
-      <div className=" mx-auto max-w-screen-xl px-4 py-12 flex justify-between items-center">
+      <div className=" mx-auto max-w-screen-xl px-4 pt-8 gap-6 sm:gap-0 sm:py-12 flex flex-col sm:flex-row justify-between items-center">
         <div className="max-w-xl ">
           <div className="text-mynavy">
             <Head h1="Make Your Look more" h2="Perfect" />
```
Comment (pooranjoyb) [inline src/pages/Home/Home.tsx:15]: Make changes in the `Head` component for compatible font-size in smaller devices.

**Review by pooranjoyb — CHANGES_REQUESTED** [review]

File: src/pages/Home/Home.tsx (hunk 2 above)
Comment (pooranjoyb) [reply src/pages/Home/Home.tsx:15]: Also resolve font-size for for all the texts in mobile devices.

**Comment by Sid-80** [comment]
@pooranjoyb Footer image is not visible in mobile view will be fine or should make it visible ?

**Comment by Sid-80** [comment]
I converted it into jpg and now it is visible

**Comment by Sid-80** [comment]
@pooranjoyb Everything is set! You can test it

**Comment by pooranjoyb** [comment]
> I converted it into jpg and now it is visible

Still Footer is not visible dude. Please try to fix that :)

image

**Comment by Sid-80** [comment]
Done!

**Comment by pooranjoyb** [comment]
> Done!

Awesome! Last bug before I merge it!

image

Make sure the search bar is below the `Heading` in mobile devices! @Sid-80

**Review by pooranjoyb — APPROVED** [review]

**Comment by Sid-80** [comment]
Done!

**Comment by Mahitej28** [comment]
@pooranjoyb everything looks good to me I am merging it to dev

Good job @Sid-80 !

**Pull request merged**